lib/trento/application/usecases/sap_systems/health_summary_service.ex

```diff
@@ -60,16 +64,24 @@ defmodule Trento.SapSystems.HealthSummaryService do
     |> HealthService.compute_aggregated_health()
   end
 
-  @spec compute_clusters_health(instance_list) :: Health.t()
-  defp compute_clusters_health(instances) do
+  @spec compute_cluster_health(instance_list, [ClusterType.t()]) :: Health.t()
+  defp compute_cluster_health(instances, cluster_types) do
     instances
     |> reject_unclustered_instances()
     |> clusters_from_instance()
-    |> keep_only_hana_scale_up_clusters()
+    |> filter_by_cluster_type(cluster_types)
```

Comment:
If it was done from scratch, I would left the simple `reject/filter` enum functions here, without hiding them in functions, but just followed what we had.
If you want I can put them here directly. Wdyt?

Reply:
Nevermind, i have changed the code XD